.gitignore

```diff
@@ -3,7 +3,7 @@
 *.iml
 .env
 .DS_Store
-venv/
+.venv/
```

Comment:
The `.venv/` entry is correctly included to ignore virtual environment directories. Ensure that this entry matches the actual directory name used in your project if it's different.